language/go/config.go

```diff
@@ -372,6 +446,17 @@ Update io_bazel_rules_go to a newer version in your WORKSPACE file.`
 				log.Printf("Found RULES_GO_VERSION %s. Minimum compatible version is %s.\n%s", gc.rulesGoVersion, minimumRulesGoVersion, message)
 			}
 		}
+		repoNamingConvention := map[string]namingConvention{}
+		for _, repo := range c.Repos {
+			if repo.Kind() == "go_repository" {
+				if nc, err := namingConventionFromString(repo.AttrString("build_naming_convention")); err == nil {
+					repoNamingConvention[repo.Name()] = nc
+				} else {
+					log.Printf("%v\n", err)
```

Comment:
More context needed here:

```
log.Printf("in go_repository named %q: %v", repo.Name(), err)
```

No need for a newline at the end of `log.Printf` messages: one is added automatically.